connectors/byoc.py

```diff
@@ -575,6 +552,7 @@ async def prepare(self, config):
             raise ConnectorUpdateError(
                 f"Could not update service type/configuration for connector {self.id}"
             )
+        await self.reload()
```

Comment:
What's the reason of adding the reload here?

Reply:
So the `connector` object has the latest `service_type` and `configuration`.